ogre2/CONTROL

```diff
@@ -1,5 +1,5 @@
 Source: ogre2
-Version: 2.0.9999~20180616~06a386f
+Version: 2.1.9999~20210410~71fbebd
```

Comment:
```suggestion
Version: 2.0.9999~20210410~71fbebd
```
When creating a version for a package that will be released as 2.1 we use 2.0.9999 so if at some point the real stable package appear as 2.1 the packages that we built before that one do not take precedence over it.

The timestamp should serve vcpkg to understand that our package is newer than the current one.